module/spl/spl-kmem-cache.c

```diff
@@ -966,16 +985,17 @@ spl_kmem_cache_create(char *name, size_t size, size_t align,
 		} else {
 			/*
 			 * All other objects are considered large and are
-			 * placed on vmem backed slabs.
+			 * placed on kvmem backed slabs.
 			 */
+			//skc->skc_flags |= KMC_KVMEM;
```

Comment:
This line should be deleted. We do not keep commented out code in the repository. The commit history is for that. If this is being left in place as some kind of note, please use an English language comment that will be more meaningful to other contributors.

Reply:
Was debugging, too. Removed